src/runtime-rs/crates/resource/src/network/network_model/route_model.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::process::Command;
+
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+
+use super::{NetworkModel, NetworkModelType};
+use crate::network::NetworkPair;
+
+#[derive(Debug)]
+pub(crate) struct RouteModel {}
+
+impl RouteModel {
+    pub fn new() -> Result<Self> {
+        Ok(Self {})
+    }
+}
+
+#[async_trait]
+impl NetworkModel for RouteModel {
+    fn model_type(&self) -> NetworkModelType {
+        NetworkModelType::Route
+    }
+
+    async fn add(&self, pair: &NetworkPair) -> Result<()> {
+        let tap_name = &pair.tap.tap_iface.name;
+        let virt_name = &pair.virt_iface.name;
+
+        //setup rule
+        let output = Command::new("ip")
```

Comment:
Should we switch it to tokio Command, The output()  may block the async runtime

Reply:
Thanks for your review, change to `tokio::process::Command`